### ISSUE TITLE
LocalNotification not working on reactnative 0.59

### ISSUE THREAD

**Issue by mohstarsos1** [zo0r/react-native-push-notification#1020]
hello
i created the new react native(v 0.59) and install this library but PushNotification.localNotification not working!

Please help me...TNX

**Comment by abarisic86** [comment]
Can you update your description with additional details like the error you are getting or how you concluded that there is an issue inside this library? (or any other details that can help)

**Comment by Ravi448** [comment]
Hi, 
I'm also facing the same issue. 
I followed every steps as well as build is successful, but there's no localNotification!!
I searched for look arounds and the suggested fix is to downgrade `gradle:3.1.1`, which is not possible for ongoing project.
There's no error and no build failure. 

`"react": "16.8.3"`
  `"react-native": "0.59.8"`
`  "react-native-push-notification": "^3.1.3"`

Android config:
`'com.android.tools.build:gradle:3.4.1'`
Wrapper:
`gradle-5.1.1-all.zip`

I'm not sure, but it might be package issue.
